keywording/text.xml

```diff
@@ -280,6 +280,22 @@ blank or <c>-</c> sanity-check, so please fix this if it occurs <d/> or ask for
 help to do so.
 </p>
 
+<note>
+If NATTkA gives a failed sanity check result in a comment or otherwise
+sets the <c>sanity-check</c> field to <c>-</c>, you must clear it by
+setting it to blank in order for NATTkA to consider e.g. changes
+in the package list made in response to any issues. Failure to do
+this will result in no change occurring. You should also reset
+the field if you make any changes to the package list to ensure
+they are read by NATTkA.
+</note>
+
+<note>
+If you are changing the package list of a bug after it has been processed,
+<d/> i.e. after NATTkA has CCed arches <d/> you should un-CC all arches so that
+NATTkA CCs the correct new set of arches, in addition to the advice above.
+</note>
+
```

Comment:
That's now three `<note>`s in a row. I wonder if the first one could be changed to a regular paragraph? It's length suggests that it should.

Reply:
I think this should be obsolete now.